classes/module-signing.bbclass

```diff
@@ -10,17 +10,35 @@ INHIBIT_PACKAGE_STRIP = "1"
 export HOST_EXTRACFLAGS = "${BUILD_CFLAGS} ${BUILD_LDFLAGS}"
 
 # Set KERNEL_MODULE_SIG_KEY in local.conf to the filepath of a private key
-# for signing kernel modules.  If unset, signing can be done offline.
+# for signing kernel modules.
```

Comment:
I think it can still be valid to have KERNEL_MODULE_SIG_KEY unset for offline signing.

Reply:
I discarded that use-case to unblock the rest, I'll amend and add the `bbfatal`. `KERNEL_MODULE_OFFLINE_SIGNING` sounds fine, it will require changes in https://github.com/OpenXT/xenclient-oe/pull/1395 for `do_image_qa`.